### DISCUSSION TITLE
Update the roman numbering of the pages in the UMinho template

### DISCUSSION THREAD

**Discussion by Marcelo-Alves7** [joaomlourenco/novathesis#429] [Q&A]
Hello,

I'm using the 7.0.0 branch in Overleaf with the Uminho template.

According to the rules, except for the title page, which must not be numbered but must be counted, all remaining pages of text must be numbered in Roman numerals.
Therefore, after the title page comes to the statement regarding the conditions of use of the work by partners, and this page should be numbered with the number 2 (Roman number), but at the moment only 1 (Roman number) appears.

How can I change?
Thanks

**Comment by joaomlourenco** [comment]
Fixed in the most recent version.
You may update only the files:

- `novathesis.cls`
- `NOVAthesisFiles/Schools/uminho/uminho-defaults.ldf`
